requirements.txt

```diff
@@ -18,80 +18,82 @@ colorama==0.4.3
 constant-sorrow==0.1.0a9
 constantly==15.1.0
 cryptography==2.9.2
-cytoolz==0.10.1 ; implementation_name == 'cpython'
-dateparser==0.7.4
-eth-abi==2.1.1
-eth-account==0.5.2
-eth-bloom==1.0.3
+cytoolz==0.10.1; implementation_name == 'cpython'
+dateparser==0.7.6; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+eth-abi==2.1.1; python_version >= '3.6' and python_version < '4'
+eth-account==0.5.2; python_version >= '3.6' and python_version < '4'
+eth-bloom==1.0.3; python_version >= '3.5' and python_full_version != '3.5.2' and python_version < '4'
 eth-hash[pycryptodome]==0.2.0
 eth-keyfile==0.5.1
 eth-keys==0.3.3
 eth-rlp==0.1.2
 eth-tester==0.5.0b1
-eth-typing==2.2.1
-eth-utils==1.9.0
+eth-typing==2.2.1; python_version >= '3.5' and python_version < '4'
+eth-utils==1.9.0; python_version >= '3.5' and python_full_version != '3.5.2' and python_version < '4'
 flask-sqlalchemy==2.4.3
 flask==1.1.2
 hendrix==3.3.0
-hexbytes==0.2.1
-humanize==2.4.0
+hexbytes==0.2.1; python_version >= '3.6' and python_version < '4'
+humanize==2.5.0; python_version >= '3.5'
 hyperlink==19.0.0
-idna==2.9
+idna==2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 incremental==17.5.0
-ipfshttpclient==0.4.13.2
-itsdangerous==2.0.0a1
-jinja2==3.0.0a1
+ipfshttpclient==0.6.0.post1; python_version >= '3.5'
+itsdangerous==1.1.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+jinja2==2.11.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 jsonschema==3.2.0
+lmdb==0.98
 lru-dict==1.1.6
-markupsafe==2.0.0a1
-marshmallow==3.6.1
+markupsafe==1.1.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+marshmallow==3.7.0
 maya==0.6.1
 msgpack-python==0.5.6
-multiaddr==0.0.9
+msgpack==1.0.0
```

Comment:
@tuxxy I guess this is what @cygnusv was referring to regarding different versions of msgpack - seems msgpack-python is now deprecated - https://pypi.org/project/msgpack-python.

Reply:
Do we need to remove the `msgpack-python` dependency?

Reply:
I don't really know. I'd like to avoid making more dependency changes in this PR, though because of the testing we've performed with it. Perhaps we start another dependency update PR into `main` for this?

Reply:
A follow-up issue makes sense. Filed #2171

Reply:
Sounds good, thanks!